src/web3py/extensions/lido_validators.py

```diff
@@ -172,6 +173,14 @@ def merge_validators_with_keys(keys: list[LidoKey], validators: list[Validator])
 
         return lido_validators
 
+    @staticmethod
+    def calculate_pending_deposits_sum(lido_validators: list[LidoValidator]) -> int:
+        return sum(
+            MIN_ACTIVATION_BALANCE
```

Comment:
I think it makes sense to leave a note that it is currently true for Lido, but this may be subject to change once we implement compounding.